src/EnergyPlus/PlantLoopHeatPumpEIR.hh

```diff
@@ -197,6 +197,97 @@ namespace EIRPlantLoopHeatPumps {
         void oneTimeInit(EnergyPlusData &state) override;
     };
 
+    struct EIRFuelFiredHeatPump : public EIRPlantLoopHeatPump
```

Comment:
Up above, `EIRPlantLoopHeatPump` is declared as 
`struct EIRPlantLoopHeatPump : public EnergyPlus::BasePlantLoopHeatPump`

`BasePlantLoopHeatPump` is in [WaterToWaterHeatPumps.hh]() which is simply
```
struct BasePlantLoopHeatPump : PlantComponent
{
    ~BasePlantLoopHeatPump() = default;
};
```
Seems like we can delete this entire header and reference `PlantComponent` directly here?

Reply:
Can't we do this and skip `BasePlantLoopHeatPump`? 
`struct EIRPlantLoopHeatPump : PlantComponent`

Reply:
Thanks! I see the point now. It looks that the EIRPlantLoopHeatPump can skip `BasePlantLoopHeatPump` refence and directly inherit from `PlantComponent`.

Also, since `BasePlantLoopHeatPump` has no further definition other than in `WaterToWaterHeatPumps.hh`; and there is not other references about the `BasePlantLoopHeatPum` after the first change, the definition and the `WaterToWaterHeatPumps.hh` header file does seems to redundant now.